src/style.css

```diff
@@ -1,3 +1,88 @@
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: 'Roboto', Arial, sans-serif;` ensures that if 'Roboto' is not available, Arial will be used as a fallback.